src/ContainerIntegration.jsx

```diff
@@ -98,7 +98,10 @@ const ContainerIntegration = ({ container, localImages }) => {
     const volumes = renderContainerVolumes(container.Mounts);
 
     const image = localImages.filter(img => img.Id === container.Image)[0];
-    const env = <ContainerEnv containerEnv={container.Config.Env} imageEnv={image.Env} />;
+    let env = null;
+    // Podman allows one to remove an image while it has a container attached
```

Comment:
To be specific, it will *untag* the image, but the layers stay until they aren't referenced any more.